resources/css/search.css

```diff
@@ -9,8 +9,13 @@
 }
 
 .searchbox-top {
-  background-color: var(--embed-color);
-  border-radius: 4px;
+  height: 22px;
```

Comment:
22px feels a bit cramped. We can probably up this to 30px without impacting the navbar height.